robot/rover/comms/src/CommandCenter.cpp

```diff
@@ -9,21 +9,38 @@ namespace internal_comms
     Command* CommandCenter::processCommand() const
     {
         uint8_t commandID = Serial.read();
-        size_t bytesToRead = Serial.read();
-        uint8_t* buffer = (uint8_t*) malloc(sizeof(uint8_t) * bytesToRead);
-        uint8_t bytesRead = (uint8_t) Serial.readBytes((char*)buffer, bytesRead);
+        uint8_t deviceSending = Serial.read();
+        uint8_t deviceReceiving = Serial.read();
+        uint16_t messageLength = readMessageSize();
```

Comment:
This is the length of the entire message though as opposed to just the argument length, is it not? If it is the argument length, can this be renamed to something more suitable like argumentLength?

Reply:
Its the message within the message 👀

Reply:
Its... Inmessaption

Reply:
Yeah, I had to reread the readMessageSize() implementation. Would be good to rename to argumentLength.

Reply:
I changed the names